upgrade/patches/43_api_token.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE {{%api_token}} (
+  apt_id int unsigned NOT NULL auto_increment,
+  apt_usr_id int unsigned NOT NULL,
+  apt_created datetime NOT NULL,
+  apt_status varchar(10) NOT NULL default 'active',
+  apt_token varchar(32) NOT NULL,
+  PRIMARY KEY (apt_id),
+  KEY (apt_token)
```

Comment:
Having an auto increment ID makes life easier and pretty much every other table in eventum uses it. 

As far as excluding the token from the SQL query, what is your reason for this? Since we aren't hashing the token it shouldn't be vulnerable to a timing attack.